lib/ansible/modules/windows/win_service.py

```diff
@@ -113,6 +113,20 @@
     - A newly created service will default to C(LocalSystem).
     type: str
     version_added: '2.3'
+  recovery_reset_interval:
+    description:
+      - Reset failure counter after this number of seconds.
+    type: int
+    default: 1
```

Comment:
Why not make it the default Windows uses?

Reply:
I'll double check what the default is and change this.

Reply:
There's no default in Windows, actually the GUI shows "0".
And you have to set it when setting actions.

Reply:
It seems the value is in days at least in win10 / server 16 and the defautl is 1 day.
in hex it is saved in milliseconds so 1 day = 80510100